backend/space/middleware.ts

```diff
@@ -42,7 +42,7 @@ const isPlaceAlreadyExists = async (req: Request, res: Response, next: NextFunct
  * Checks if place_id in req.params exists
  */
 const isPlaceExists = async (req: Request, res: Response, next: NextFunction) => {
-    const place_id: string = req.params.place_id
```

Comment:
i believe this should still be params. may need to ignore changes on this branch for `backend/space/middleware.ts` for merge